tests/plugins/pastefromword/parsestyles.js

```diff
@@ -12,6 +12,14 @@
 		bender.tools.testInputOut( name, function( styles, expected ) {
 			var tested = CKEDITOR.plugins.pastefromword.styles.inliner.parse( styles );
 
+			// In CKEDITOR.plugins.pastefromword.styles.inliner.parse#createIsolatedStylesheet function
+			// somehow Edge camelcases the selectors so we need to lowercase it (#962).
```

Comment:
This should be troubleshooted, and reported to upstream, sounds like a serious issue. Might be coming from here: https://github.com/ckeditor/ckeditor-dev/blob/ea41a52/plugins/pastefromword/filter/default.js#L810-L820

In which case it would be crucial problem for our styles inlining.

Luckily (😌) I smell something fishy is happening in our codebase 🙂 See the expected value in: https://github.com/ckeditor/ckeditor-dev/blob/ea41a52/tests/plugins/pastefromword/parsestyles.html#L7-L12

It **should be camelcased** so Edge provides a correct value, the problem is why expected value gets lowercased?

After I have logged it it turns out that chrome also puts the expected value as lowercase, which doesn't make sense. I'll leave it here for you.

Reply:
I extracted it to a separate issue as it seems to be quite odd behaviour #1042.